proto/cloud_teams.proto

```diff
@@ -16,7 +16,6 @@ enum MessageType {
 
 message Message {
   string teamId = 1;
-  string activityId = 2;
```

Comment:
It's ok to remove the field as we updated the plugin contract version recently. Is that correct?

Reply:
it's about cloud_teams, so it is not related, it's only about control-plane <-> agent communication. They should work properly as this field was not used anyway 👍